src/tezos_interop/tezos_bridge.ml

```diff
@@ -50,6 +53,42 @@ module Transaction = struct
     | _ -> Error "invalid status"
 end
 
+module Storage = struct
+  type kind = Storage
+  let kind_to_yojson Storage = `String "storage"
```

Comment:
Noob question: would it be the same to do:
```ocaml
type kind = Storage [@name storage]
  [@@deriving to_yojson]
```

Reply:
@gsebil08 nope, also tried that, TLDR variants are encoded as `[header, ...values]` so it endup as `["storage"]`